### PR TITLE
[REF] 8.0 mgmtsystem nonconformity analytic

### PR DESCRIPTION
**Pull request by llacroix**
Marked as installable and no changes were required to make it work. Can be moved to Version 8

Depends on:
- [x] #25

### CODE REVIEW

**Comment by bwrsandman** [comment]
Please rename `mgmtsystem_nonconformity.xml` as `mgmtsystem_nonconformity_view.xml`

**Comment by llacroix** [comment]
Wouldn't it be better to put xml views in a folder named `views`. I think I saw that kind of structure in different modules.

**Comment by bwrsandman** [comment]
@llacroix That isn't a requirement, but it is a plus!

**Comment by max3903** [comment]
@llacroix you are missing a `__init__.py` file in the `models` directory.

**Comment by bwrsandman** [comment]
This PR needs to be rebased

**Comment by llacroix** [comment]
Rebased

File: mgmtsystem_nonconformity_analytic/__openerp__.py [hunk 1]
```diff
@@ -24,8 +24,12 @@
     "license": "AGPL-3",
     "category": "Management System",
     "description": """Add Analytic Accounts / Contracts to Non Conformities""",
```
Comment (dreispt) [inline mgmtsystem_nonconformity_analytic/__openerp__.py]: Remove and add a README.rst

File: mgmtsystem_nonconformity_analytic/__init__.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# -*- encoding: utf-8 -*-
```
Comment (dreispt) [inline mgmtsystem_nonconformity_analytic/__init__.py]: coding

**Comment by max3903** [comment]
:+1:

**Comment by dreispt** [comment]
:+1:

**Pull request merged**